contents/handbook/growth/sales/new-sales.md

```diff
@@ -4,44 +4,68 @@ sidebar: Handbook
+This is an overview for what you should actually be doing with a customer at each stage of the sales process. For details on how to manage this in our CRM, visit our [Salesforce docs](/handbook/growth/sales/crm). The steps are:
+
+1. People contact us
+2. We assign and qualify
+3. Initial demo
+4. Product evaluation
+5. Security & legal review (only if asked - skip otherwise)
+6. Commercial evaluation
+7. Closed - Won or Lost
+
+### 1. People contact us
 
-**1. People contact us**
+Most people fill in the [form](/talk-to-a-human) via the website. Some email sales@posthog.com or open a Zendesk ticket. 
 
-Most people fill in the [form](/talk-to-a-human) via the website. Some email sales@posthog.com or open a Zendesk ticket.
+### 2. We assign and qualify
 
-**2. We assign and qualify**
+Info on how leads are assigned can be [found here](https://posthog.com/handbook/growth/sales/crm#how-we-do-lead-assessment). Once you have been assigned a lead, you'll want to qualify them - don't just immediately go out and offer a demo every time. Things to consider:
 
-Cameron reviews these and decides whether to route to [self-serve](/handbook/growth/sales/crm#inbound-self-serve-pipeline), schedule a demo, or pass to Simon (if they seem Very Large).  This handoff is currently a gut feel based on company size/revenue as Very Large customers may technically be considered low ICP per our [scoring](/handbook/growth/sales/icp). 
+- How closely do they match our [ICP](/handbook/who-we-are-building-for)?
+- Which products are they looking to use? What's their use case?
+- How large is their company? Revenue? Have they raised funding? (ie. will they pay >$20k for PostHog?)
+- What is the role of the person who filled out the form?
 
-**3. Initial demo**
+Most companies add friction here by making customers jump on a call first to qualify them - we don't do this because it's annoying to the customer. However, it's totally fine to ask them questions over email in advance of the demo to make sure you're making the best use of their time - just be specific. A few clarifying questions is fine, a 30 question survey is not. 
 
-Some general principles:
-- Be genuinely helpful - our customers usually already have specific problems in mind, so help them solve those first rather 
-- Get to the point - our ICP is technical and particularly sensitive to sales BS
-- Don't [promise](/handbook/growth/sales/overview#enterprise-customers) things we can't/don't want to deliver
-- Get into the actual demo as quickly as possible - you should spend most of your time here, not presenting PostHog's vision
+If you're pretty sure that they should be qualified out, you should still be helpful over email - some customers just use the form to get in touch and don't want to actually have a demo (e.g. they have a billing question or are asking about compliance things like HIPAA.)
```

Comment:
```suggestion
If you're pretty sure that they should be qualified out, you should still be helpful over email - some customers just use the form to get in touch and don't want to actually have a demo (e.g. they have a billing question or are asking about compliance things like HIPAA.)

You can also redirect them to use the In-app support modal if they have a product-related question - this will then be routed to the right team, as well as showing them CTAs to upgrade for high priority support.
```